README.md

```diff
@@ -189,9 +189,36 @@ ALIASES
 
 _See code: [src/commands/config/view.ts](https://github.com/architect-team/architect-cli/blob/v1.13.1/src/commands/config/view.ts)_
 
+## `architect dev [CONFIGS_OR_COMPONENTS]`
```

Comment:
For future ref - this should be autogenerated by oclif during a release. See the `<!-- commands -->` block. Oclif uses that as a marker to insert generated commands.